.pre-commit-config.yaml

```diff
@@ -35,20 +35,21 @@ repos:
       rev: v0.991
       hooks:
           - id: mypy
-            additional_dependencies: ['types-PyYAML', 'types-requests', 'pydantic']
-            exclude: "tests|templates"
+            additional_dependencies: ['types-PyYAML', 'types-requests', 'pydantic', 'fastapi', 'click']
+            exclude: "tests|templates|packaging"
```

Comment:
You could also exclude `packaging` directory on the root level of the config file, unless you have several hooks that need to look into this directory

Reply:
If the above applies, you could still explicitly include this directory in the respected hooks

Reply:
No please don't . Excludes should be done per check and be as narrow as possible, so as much files as possible are checked by pre-commit for issues. For example I want to have flake8 checks of packaging/deb/data/etc/kat/keiko.gunicorn.conf.py because I want to get an error if we ever use an undefined variable in that file. Also that whole directory needs to be checked for trailing whitespace, line ending, etc.

I also don't see why packaging needs to be excluded for mypy because mypy doesn't give any error on the directory.

Reply:
Ah right, I completely forgot there were Python files! I just assumed it was mostly config and template files

Reply:
I removed the packaging exclusion from both pylint and mypy